stacker/lookups/output.py

```diff
@@ -0,0 +1,35 @@
+from collections import namedtuple
```

Comment:
Sorry, I meant a new subdirectory in: `lookups/handlers` since this directory also has the registry in it.  Probably not a big deal, just thought it'd make sense to separate them out for future readers.